docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -7,21 +7,82 @@ Fairness in Machine Learning
 Fairness of AI systems
 ----------------------
 
-AI systems can behave unfairly for a variety of reasons. Sometimes it is
-because of societal biases reflected in the training data and in the decisions
-made during the development and deployment of these systems. In other cases,
-AI systems behave unfairly not because of societal biases, but because of
-characteristics of the data (e.g., too few data points about some group of
-people) or characteristics of the systems themselves. It can be hard to
-distinguish between these reasons, especially since they are not mutually
-exclusive and often exacerbate one another. Therefore, we define whether an AI
-system is behaving unfairly in terms of its impact on people — i.e., in terms
-of harms — and not in terms of specific causes, such as societal biases, or in
-terms of intent, such as prejudice.
-
-**Usage of the word bias.** Since we define fairness in terms of harms
-rather than specific causes (such as societal biases), we avoid the usage of
-the words *bias* or *debiasing* in describing the functionality of Fairlearn.
+AI systems can behave unfairly for a variety of reasons. These reasons seldom
+include bad intentions on the part of the teams responsible for developing and
+deploying these systems. Some AI systems behave unfairly because of societal
+biases that are reflected in the datasets used to train them or reflected in
+the assumptions or decisions made (either explicitly or implicitly) by teams
+during the AI development and deployment lifecycle. Other systems behave
+unfairly not because of societal biases, but because of characteristics of the
+datasets used to train them (e.g., too few data points about some group of
+people) or characteristics of other system components. It can be hard to
+distinguish between these reasons, especially because these reasons are not
+mutually exclusive and often exacerbate one another. Therefore, we define
+whether an AI system is behaving unfairly in terms of its impact on people —
+i.e., in terms of harms — and not in terms of specific causes, such as
+societal biases. Because we define unfairness in this way, we avoid using the
+word *bias* to refer to the unfair behavior of an AI system. Also *bias* is
+ambiguous and means different things to different communities – e.g.,
+statistical bias vs. societal biases. For this reason, we talk about
+*fairness issues* or *fairness-related harms* rather than *bias*, unless
+we are very specifically referring to societal biases (or statistical bias or
+some other definition of bias). Given the many complex sources of unfairness,
+it is not possible to fully *debias* a system or to guarantee fairness; the
+goal is to detect and to mitigate fairness-related harms as much as possible.
+For this reason, don’t use words like *debias*, *unbiased*, *solve* – they set
```

Comment:
This is a sudden switch to the imperative, when the rest has been "we do this" or "we don't do that"

Reply:
Miro will rewrite this section of the user guide in a PR of his own, so I'll just put the list into the contributor guide. Reverting the last few commits and adjusting the lists according to earlier feedback.